driverapi/driverapi.go

```diff
@@ -14,6 +14,9 @@ type Driver interface {
 	// Push driver specific config to the driver
 	Config(options map[string]interface{}) error
 
+	// Capabilities of the driver
+	Capabilities() Capability
```

Comment:
IMO this should not be a separate api that can be invoked by libnetwork to get the capability every time. It should just some information that is set when the driver registers the first time. The reason for that is that this information should be an init time data. If we keep asking for this information every time we are saying that this information can dynamically change which I assume we cannot handle in libnetwork. For example if the driver said it is global scoped while creating the network and said local scope when deleting the network it would cause confusion.